contribute/hacker-guide.md

```diff
@@ -349,7 +302,7 @@ This means your change is backward or forward binary incompatible with the speci
 
 Now to make sure that my fix doesn't break anything I need to run the test suite using the `partest` tool we wrote to test Scala.
 Read up [the partest guide](partest-guide.html) to learn the details about partest, but in a nutshell you can either
-run `ant test` to go through the entire test suite (30+ minutes) or use wildcards to limit the tests to something manageable:
+run `sbt test` to go through the entire test suite (30+ minutes) or use wildcards to limit the tests to something manageable:
```

Comment:
not sure what you mean by wildcard filtering, but it's true that the `partest` task within sbt is not the same as the `test/partest` script. the sbt task may not implement all the same options (https://github.com/scala/scala/blob/2.12.x/build.sbt#L824-L827 and https://github.com/scala/scala/blob/2.12.x/project/PartestUtil.scala)